Makefile

```diff
@@ -6,7 +6,15 @@
 # %LICENSE%
 #
 
-VERSION="3.3 rev: $(shell git rev-parse --short HEAD)"
+# DESC will be in format v<VERSION> <#COMMITS SINCE TAG> g<CURRENT COMMIT HASH>[ dirty]
+DESC := $(subst -, ,$(shell git describe --dirty --abbrev=8 --tags --long --match "v[[:digit:]].[[:digit:]]*.[[:digit:]]*"))
```

Comment:
there was a talk that the build should also work on sources exported from git (so no `git` binary, no tags). Please provide some fallback (dummy values with fixme might be ok for now, in the future we might need to save version to some file inside repo on release)